contrib/platform/src/com/sun/jna/platform/win32/WinUser.java

```diff
@@ -2001,4 +2003,83 @@ public String toString() {
      * or 22 kHz PCM.
      */
     public int CF_WAVE = 12;
+
```

Comment:
Please correct the indention. There is not "one right" indention style for JNA, but in most cases 4 spaces per indention level is correct. Tabs are normally not used.